test/DataClasses/testDataClasses.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+#import unittest
+#sys.path.insert(0, '/')
+#import owner
+import unittest, sys
+sys.path.append('../../')
+from src.DataClasses.owner import Owner
+from src.DataClasses.property import Property
+from src.DataClasses.link import Link
+
+class TestDataClasses(unittest.TestCase):
+	def test_owner(self):
+		testfirstName = 'John'
+		testlastName = 'Doe'
+		testemail = 'john@doe.com'
+		testphone = 5555555555
+		testsocialMedia = {
+			'Facebook': 'facebook_url',
+			'LinkedIn': 'linkedIn_url',
+			'Instagram': 'instagram_url',
+			'Twitter': 'twitter_url'
+		}
+		testOwner = Owner(testfirstName, testlastName, testemail, testphone, testsocialMedia)
+		self.assertEquals(testOwner.firstName, testfirstName)
+		self.assertEquals(testOwner.lastName, testlastName)
+		self.assertEquals(testOwner.email, testemail)
+		self.assertEquals(testOwner.phone, testphone)
+		self.assertEquals(testOwner.socialMedia['Facebook'], 'facebook_url')
+		self.assertEquals(testOwner.socialMedia, testsocialMedia)
```

Comment:
This is a good test.  It not only tests functionality, but gives an example of how to use the class.

Reply:
If you added a __eq__() to the Owner class, you could have just one assert in your test comparing the created object to a test object.